packages/vaadin-combo-box/src/vaadin-combo-box-dropdown-wrapper.js

```diff
@@ -50,24 +50,9 @@ class ComboBoxDropdownWrapperElement extends PolymerElement {
               box-shadow: 0 0 0 white;
             }
           </style>
-          <div id="scroller" on-click="_stopPropagation">
-            <iron-list id="selector" role="listbox" items="[[_getItems(opened, _items)]]" scroll-target="[[_scroller]]">
-              <template>
-                <vaadin-combo-box-item
-                  on-click="_onItemClick"
-                  index="[[__requestItemByIndex(item, index, _resetScrolling)]]"
-                  item="[[item]]"
-                  label="[[getItemLabel(item, _itemLabelPath)]]"
-                  selected="[[_isItemSelected(item, _selectedItem, _itemIdPath)]]"
-                  renderer="[[renderer]]"
-                  role$="[[_getAriaRole(index)]]"
-                  aria-selected$="[[_getAriaSelected(_focusedIndex,index)]]"
-                  focused="[[_isItemFocused(_focusedIndex,index)]]"
-                  tabindex="-1"
-                  theme$="[[theme]]"
-                ></vaadin-combo-box-item>
-              </template>
-            </iron-list>
+
+          <div id="scroller" on-click="_stopPropagation" style="min-height: 1px">
```

Comment:
Not a blocker: I think we can change this div to become a web component e.g. `vaadin-combo-box-scroller` and then move styles above inside of it, to make getting rid of `template` easier when changing it to `renderer` later (so that items would be in light DOM). Feel free to do it in a separate PR.